Sources/Formworks/Design System/FWStyle.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  FWStyle.swift
+//  
+//
+//  Created by Cassia Aparecida Barbosa on 29/10/20.
+//
+
+import Foundation
+
+/// The form's color style.
+public enum FWStyle {
+	/// The default light color style of a form.
+	case light
+	
+	/// The default dark color style of a form.
+	case dark
+	
+	/// A custom color style of a form.
+	case custom(FWStyleSpecification)
```

Comment:
How will be the custom color scheme be implemented? I really forgot during the day on what we discussed about the custom colors.